src/Storefront/Resources/app/storefront/src/plugin/navbar/navbar.plugin.js

```diff
@@ -31,7 +31,9 @@ export default class NavbarPlugin extends Plugin {
         this._topLevelLinks.forEach(el => {
             el.addEventListener(openEvent, this._toggleNavbar.bind(this, el));
             el.addEventListener(closeEvent, this._toggleNavbar.bind(this, el));
-            el.addEventListener(clickEvent, this._navigateToLinkOnClick.bind(this, el));
+            if (el.dataset?.flyoutMenuWithoutTarget !== 'yes') {
```

Comment:
Instead of introducing this new `data-flyout-menu-without-target="yes"` couldn't we check if the given element has an href? Inside `_navigateToLinkOnClick` the href is directly used. href should be `null` when the attribute cannot be found.


```suggestion
            if (el.getAttribute('href') !== null) {
```

Reply:
Good point. Will change accordingly :slightly_smiling_face: